saltinit.py

```diff
@@ -7,6 +7,17 @@
 
 async def main():
     futures = []
+    salt_uid = os.environ['SALT_UID'] if 'SALT_UID' in os.environ else '100'
```

Comment:
when i built the containers with your PR and tried to run it i see this error:

```
Setting salt UID to 100
Traceback (most recent call last):
  File "/usr/local/bin/saltinit", line 71, in <module>
    loop.run_until_complete(main())
  File "/usr/local/lib/python3.7/asyncio/base_events.py", line 583, in run_until_complete
    return future.result()
  File "/usr/local/bin/saltinit", line 14, in main
    usermod = await asyncio.create_subprocess_exec('usermod', '-u', salt_uid, 'salt')
  File "/usr/local/lib/python3.7/asyncio/subprocess.py", line 217, in create_subprocess_exec
    stderr=stderr, **kwds)
  File "/usr/local/lib/python3.7/asyncio/base_events.py", line 1540, in subprocess_exec
    bufsize, **kwargs)
  File "/usr/local/lib/python3.7/asyncio/unix_events.py", line 190, in _make_subprocess_transport
    **kwargs)
  File "/usr/local/lib/python3.7/asyncio/base_subprocess.py", line 37, in __init__
    stderr=stderr, bufsize=bufsize, **kwargs)
  File "/usr/local/lib/python3.7/asyncio/unix_events.py", line 775, in _start
    universal_newlines=False, bufsize=bufsize, **kwargs)
  File "/usr/local/lib/python3.7/subprocess.py", line 800, in __init__
    restore_signals, start_new_session)
  File "/usr/local/lib/python3.7/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'usermod': 'usermod'
```

Reply:
Ah, I removed a little bit too much in the apk install command. I added the required `shadow` package back